h/services/group_create.py

```diff
@@ -126,7 +126,7 @@ def _create(self, name, userid, type_flags, scopes, **kwargs):
         # self.publish() or `return group`.
         self.db.flush()
 
-        group.members.append(group.creator)
+        group.memberships.append(GroupMembership(user=group.creator))
```

Comment:
I don't have to pass `group=group` to `GroupMembership()`, SQLAlchemy will figure out the group automatically because I'm appending the `GroupMembership` to `group.memberships`.